docs/source/modin/supported/index_supported.rst

```diff
@@ -65,9 +65,9 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | DataFrame method            | Snowpark implemented? (Y/N/P/D) | Missing parameters               | Notes for current implementation                   |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``all``                     | N                               |                                  |                                                    |
+| ``all``                     | P                               |                                  | ``N`` for non-integer/boolean types.               |
```

Comment:
```suggestion
| ``all``                     | P                               |                                  | ``N`` for non-numeric types.               |
```

Reply:
I wrote non-integer/boolean because float types don't work

Reply:
why float does not work? It works in native pandas though.

Reply:
It's because floats cannot be cast to boolean values in Snowflake (this is what the existing Series query compiler method does). I'm thinking maybe trying to round them or using some `IFF` statement might get around this? I'll try that out.

Reply:
if `pd.Series([1.1, 0.1, 3]).all()` does not work then we could fix it too.